README.md

```diff
@@ -4,5 +4,5 @@ PrairieLearn is an online problem-driven learning system for creating homeworks
 
 [![GitHub Actions build status](https://github.com/PrairieLearn/PrairieLearn/actions/workflows/main.yml/badge.svg)](https://github.com/PrairieLearn/PrairieLearn/actions/workflows/main.yml) [![License](https://img.shields.io/github/license/PrairieLearn/PrairieLearn.svg)](https://github.com/PrairieLearn/PrairieLearn/blob/master/LICENSE)
 
-Documentation website: [https://prairielearn.readthedocs.io/](https://prairielearn.readthedocs.io/)
+Documentation website: [https://prairielearn.readthedocs.io/](https://prairielearn.readthedocs.io/)\
```

Comment:
Nit, let's just use linebreaks, the trailing slash is kinda non-obvious:
```suggestion
Documentation website: [https://prairielearn.readthedocs.io/](https://prairielearn.readthedocs.io/)

```

Reply:
Yes, blank lines in between cause a paragraph separation, subsequent lines with no trailing slash run into the same formatted line, and the trailing lash inserts a newline in the same paragraph. However, if you end a line with _two_ spaces (instead of a slash), that will also induce a newline.

Reply:
I avoid the two trailing spaces method just because I sometimes use a "remove trailing spaces" macro in my editor.

Reply:
I have a strong dislike of the trailing double space because (1) you can't see it at all and (2) I typically like having my editor auto-trim trailing whitespace whenever possible.

Reply:
Separate paragraphs are totally fine. Alternatively, a bulleted list would also be good 👍🏻